js/MVP/views/moteinnkallelse/Moteinnkallelse.tsx

```diff
@@ -51,21 +51,21 @@ const title = (type: string): string => {
 const Moteinnkallelse = (): ReactElement => {
   const brev = useBrev();
 
-  const brevHead = Array.isArray(brev.data) ? brev.data[0] : null;
-  const { tid, uuid, brevType, document, lestDato } = brevHead;
-
   if (brev.isLoading) {
     return <AppSpinner />;
   }
 
   if (brev.isError) {
     return (
-      <DialogmoteContainer title={title('')} breadcrumb={innkallelseBreadcrumb(title(brevType))} displayTilbakeknapp>
+      <DialogmoteContainer title={title('')} breadcrumb={innkallelseBreadcrumb(title(''))} displayTilbakeknapp>
         <FeilAlertStripe />
       </DialogmoteContainer>
     );
   }
 
+  const brevHead = Array.isArray(brev.data) ? brev.data[0] : null;
```

Comment:
Trenger man isArray? Ellers, dersom fallback inntreffer, vil det da kræsje på linje 67?

Reply:
Testet i console nå. "VM99:1 Uncaught TypeError: Cannot destructure property 'tid' of 'test' as it is null.
    at <anonymous>:1:9"

Reply:
> 

Bra catch, det glemte jeg om. Sletter her også.

Reply:
> @AudunSorheim AudunSorheim 1 hour ago
Trenger man isArray? Ellers, dersom fallback inntreffer, vil det da kræsje på linje 67?

Bra catch, det glemte jeg om. Sletter her også.

Reply:
> @AudunSorheim AudunSorheim 38 minutes ago
Testet i console nå. "VM99:1 Uncaught TypeError: Cannot destructure property 'tid' of 'test' as it is null.
at :1:9"

Hva mener du her?  Vennligst presiser

Reply:
Jeg bare testet i chrome console om destructuring av null vil kræsje. Og det gjorde det :)